prep-javascript-objects/main.js

```diff
@@ -0,0 +1,14 @@
+const person = {
+  firstName: 'Gabrielle Ramos',
+  lastName: 'Ramos',
+  hobby: 'baking',
+  job: 'student',
+  previousJob: 'sales'
+};
+console.log(person);
+var fullName = ' The persons name is: Gabrielle Ramos';
```

Comment:
We can construct the person's full name using dot notation to get the `firstName` and `lastName` properties.

```js
var fullName = person.firstName + ' ' + person.lastName;
```